src/pages/profile/components/Temperature/index.tsx

```diff
@@ -0,0 +1,63 @@
+import informationLine from '@/assets/information-line.png';
+import smileFace from '@/assets/smiling-face.png';
+import { Popover, PopoverContent, PopoverTrigger } from '@/components/Popover';
+import { cn } from '@/utils/cn';
+
+interface TemperatureProps {
+  value: number;
+}
+const Temperature = ({ value = 50.3 }: TemperatureProps) => {
+  const temperatureRange = (value: number) => {
+    if (value < 12.5)
+      return { text: 'text-manner-accent7', background: 'bg-manner-accent7' };
+    if (value <= 30)
+      return { text: 'text-manner-accent6', background: 'bg-manner-accent6' };
+    if (value <= 36.5)
+      return { text: 'text-manner-accent5', background: 'bg-manner-accent5' };
+    if (value <= 50.5)
+      return { text: 'text-manner-accent4', background: 'bg-manner-accent4' };
+    if (value <= 65.5)
+      return { text: 'text-manner-accent3', background: 'bg-manner-accent3' };
+    if (value <= 88)
+      return { text: 'text-manner-accent2', background: 'bg-manner-accent2' };
+    return { text: 'text-manner-accent1', background: 'bg-manner-accent1' };
+  };
+
+  return (
+    <>
+      <Popover>
+        <PopoverTrigger>
+          <span className='flex w-fit gap-1 text-xs text-gray-accent1 underline'>
+            {'시그널온도'}
+            <img src={informationLine} alt='정보' className='h-4 w-4' />
```

Comment:
등록되어 있는 아이콘이라 `@/assets`에 이미지 제거하셔도 됩니다~
```suggestion
            <Icon id='information-line' size={16} />
```

Reply:
아 아이콘을 못찾았는데.. 있는거였네요 감사하빈다